recipes/gtsam/all/conanfile.py

```diff
@@ -161,3 +161,5 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
+        if self.settings.os == "Windows" and tools.Version(self.version) >= "4.0.3":
+            self.cpp_info.system_libs = ["dbghelp"]
```

Comment:
might be better for MinGW or cross-compiling
```suggestion
            self.cpp_info.system_libs = ["DbgHelp"]
```
but not needed to change it now

Reply:
@madebr will confirm, but lowercase always work, while the opposite is not true (for mingw@linux).

Reply:
I confirm :smile: 

<details>
<summary>These are the libraries of mingw@Linux</summary>

```
$ ls /usr/x86_64-w64-mingw32/sys-root/mingw/lib/
binmode.o
cmake
crt1.o
crt1u.o
crt2.o
crt2u.o
crtbegin.o
crtend.o
CRT_fp10.o
CRT_fp8.o
CRT_glob.o
CRT_noglob.o
dllcrt1.o
dllcrt2.o
gcrt0.o
gcrt1.o
gcrt2.o
lib6to4svc.a
libaaaamon.a
libacledit.a
libaclui.a
libactiveds.a
libactxprxy.a
libadmparse.a
libadmwprox.a
libadptif.a
libadrot.a
libadsiis.a
libadsiisex.a
libadsldp.a
libadsldpc.a
libadsmsext.a
libadsnt.a
libadvapi32.a
libadvpack.a
libaelupsvc.a
libagentanm.a
libagentctl.a
libagentdp2.a
libagentdpv.a
libagentmpx.a
libagentpsh.a
libagentsr.a
libagrmco64.a
libagtintl.a
libakscoinst.a
libalrsvc.a
libamstream.a
libamstrmid.a
libapcups.a
libapphelp.a
libappmgmts.a
libappmgr.a
libaqadmin.a
libaqueue.a
libasp.a
libaspperf.a
libasycfilt.a
libatkctrs.a
libatl.a
libatmlib.a
libatmpvcno.a
libatomic.a
libatomic.dll.a
libatrace.a
libaudiosrv.a
libauthz.a
libautodisc.a
libavicap32.a
libavifil32.a
libavrt.a
libazroles.a
libazroleui.a
libbasesrv.a
libbatmeter.a
libbatt.a
libbcrypt.a
libbidispl.a
libbitsprx2.a
libbitsprx3.a
libbnts.a
libbootvid.a
libbrowscap.a
libbrowser.a
libbrowseui.a
libbrpinfo.a
libbthci.a
libbthprops.a
libbthserv.a
libbtpanui.a
libcabinet.a
libcabview.a
libcamocx.a
libcards.a
libcatsrv.a
libcatsrvps.a
libcatsrvut.a
libccfgnt.a
libcdfview.a
libcdm.a
libcdosys.a
libcertcli.a
libcertmgr.a
libcertobj.a
libcfgbkend.a
libcfgmgr32.a
libc_g18030.a
libchsbrkr.a
libchtbrkr.a
libchtskdic.a
libciadmin.a
libcic.a
libcimwin32.a
libCINTIME.a
libciodm.a
libc_is2022.a
libc_iscii.a
libclasspnp.a
libclb.a
libclbcatex.a
libclbcatq.a
libclfsw32.a
libcliconfg.a
libclusapi.a
libcmcfg32.a
libcmdial32.a
libcmpbk32.a
libcmprops.a
libcmsetacl.a
libcmutil.a
libcnbjmon.a
libcnetcfg.a
libcnvfat.a
libcoadmin.a
libcolbact.a
libcomaddin.a
libcomadmin.a
libcomcat.a
libcomctl32.a
libcomdlg32.a
libcompatui.a
libcompstui.a
libcomrepl.a
libcomres.a
libcomsetup.a
libcomsnap.a
libcomsvcs.a
libcomuid.a
libconfmsp.a
libconnect.a
libconsole.a
libcontrot.a
libcorpol.a
libcredui.a
libcrtdll.a
libcrypt32.a
libcryptdlg.a
libcryptdll.a
libcryptext.a
libcryptnet.a
libcryptsp.a
libcryptsvc.a
libcryptui.a
libcryptxml.a
libcscapi.a
libcscdll.a
libcscui.a
libcsrsrv.a
libd2d1.a
libd3d11.a
libd3d8thk.a
libd3d9.a
libd3dcompiler_33.a
libd3dcompiler_34.a
libd3dcompiler_35.a
libd3dcompiler_36.a
libd3dcompiler_37.a
libd3dcompiler_38.a
libd3dcompiler_39.a
libd3dcompiler_40.a
libd3dcompiler_41.a
libd3dcompiler_42.a
libd3dcompiler_43.a
libd3dcompiler_46.a
libd3dcompiler_47.a
libd3dcompiler.a
libd3dcsx_46.a
libd3dcsxd_43.a
libd3dcsxd.a
libd3dx10_33.a
libd3dx10_34.a
libd3dx10_35.a
libd3dx10_36.a
libd3dx10_37.a
libd3dx10_38.a
libd3dx10_39.a
libd3dx10_40.a
libd3dx10_41.a
libd3dx10_42.a
libd3dx10_43.a
libd3dx10.a
libd3dx11_42.a
libd3dx11_43.a
libd3dx11.a
libd3dx9_24.a
libd3dx9_25.a
libd3dx9_26.a
libd3dx9_27.a
libd3dx9_28.a
libd3dx9_29.a
libd3dx9_30.a
libd3dx9_31.a
libd3dx9_32.a
libd3dx9_33.a
libd3dx9_34.a
libd3dx9_35.a
libd3dx9_36.a
libd3dx9_37.a
libd3dx9_38.a
libd3dx9_39.a
libd3dx9_40.a
libd3dx9_41.a
libd3dx9_42.a
libd3dx9_43.a
libd3dx9.a
libd3dxof.a
libdanim.a
libdataclen.a
libdatime.a
libdavclnt.a
libdavcprox.a
libdbgeng.a
libdbghelp.a
libdbnetlib.a
libdbnmpntw.a
libdciman32.a
libddraw.a
libddrawex.a
libdelayimp.a
libdeskadp.a
libdeskmon.a
libdeskperf.a
libdevenum.a
libdevmgr.a
libdevobj.a
libdevrtl.a
libdfrgifps.a
libdfrgsnap.a
libdfrgui.a
libdfsshlex.a
libdgnet.a
libdhcpcsvc6.a
libdhcpcsvc.a
libdhcpmon.a
libdhcpsapi.a
libdiactfrm.a
libdigest.a
libdimap.a
libdimsntfy.a
libdimsroam.a
libdinput8.a
libdinput.a
libdirectdb.a
libdiskcopy.a
libdispex.a
libdmconfig.a
libdmdlgs.a
libdmdskmgr.a
libdmintf.a
libdmivcitf.a
libdmocx.a
libdmoguids.a
libdmserver.a
libdmutil.a
libdmvdsitf.a
libdnsapi.a
libdnsrslvr.a
libdocprop2.a
libdocprop.a
libdpnaddr.a
libdpnet.a
libdpnhpast.a
libdpnhupnp.a
libdpnlobby.a
libdpvacm.a
libdpvoice.a
libdpvvox.a
libdrprov.a
libds32gt.a
libdsauth.a
libdsdmo.a
libdsdmoprp.a
libdskquota.a
libdskquoui.a
libdsound3d.a
libdsound.a
libdsprop.a
libdsprov.a
libdsquery.a
libdssec.a
libdssenh.a
libdsuiext.a
libduser.a
libdwmapi.a
libdwrite.a
libdxdiagn.a
libdxerr8.a
libdxerr9.a
libdxgi.a
libdxguid.a
libdxtmsft.a
libdxtrans.a
libdxva2.a
libeappcfg.a
libeappgnui.a
libeapphost.a
libeappprxy.a
libefsadu.a
libels.a
libencapi.a
libersvc.a
libes.a
libesent.a
libesentprf.a
libesscli.a
libeventcls.a
libeventlog.a
libevntagnt.a
libevntrprv.a
libevr.a
libevtgprov.a
libexstrace.a
libextmgr.a
libf3ahvoas.a
libfastprox.a
libfaultrep.a
libfcachdll.a
libfde.a
libfdeploy.a
libfeclient.a
libfilemgmt.a
libFLAC++.dll.a
libFLAC.dll.a
libfldrclnr.a
libfltlib.a
libfmifs.a
libfontext.a
libfontsub.a
libframedyn.a
libfsusd.a
libftpctrs2.a
libftpmib.a
libftpsvc2.a
libfwcfg.a
libfwpuclnt.a
libfxsapi.a
libfxscfgwz.a
libfxscom.a
libfxscomex.a
libfxsdrv.a
libfxsmon.a
libfxsocm.a
libfxsperf.a
libfxsroute.a
libfxsst.a
libfxst30.a
libfxstiff.a
libfxsui.a
libfxswzrd.a
libgcc_s.a
libgcdef.a
libgdi32.a
libgdiplus.a
libgetuname.a
libglmf32.a
libglu32.a
libgmon.a
libgpedit.a
libgpkcsp.a
libgptext.a
libguitrn.a
libgzip.a
libh323msp.a
libhal.a
libhbaapi.a
libhgfs.a
libhhsetup.a
libhid.a
libhidclass.a
libhidparse.a
libhlink.a
libhmmapi.a
libhnetcfg.a
libhnetmon.a
libhnetwiz.a
libhostmib.a
libhotplug.a
libhticons.a
libhtrn_jis.a
libhttpapi.a
libhttpext.a
libhttpmib.a
libhttpodbc.a
libhtui.a
libhypertrm.a
libiasacct.a
libiasads.a
libiashlpr.a
libiasnap.a
libiaspolcy.a
libiasrad.a
libiassam.a
libiassdo.a
libiassvcs.a
libicaapi.a
libicfgnt5.a
libicm32.a
libicmp.a
libicmui.a
libicwconn.a
libicwdial.a
libicwdl.a
libicwhelp.a
libicwphbk.a
libicwutil.a
libidq.a
libieakeng.a
libieaksie.a
libiedkcs32.a
libieencode.a
libiepeers.a
libiernonce.a
libiesetup.a
libifmon.a
libifsutil.a
libigmpagnt.a
libiis.a
libiisadmin.a
libiiscfg.a
libiisclex4.a
libiisext.a
libiislog.a
libiismap.a
libiisrstap.a
libiisrtl.a
libiissuba.a
libiisui.a
libiisuiobj.a
libiisutil.a
libiisw3adm.a
libiiswmi.a
libimagehlp.a
libimekrcic.a
libimeshare.a
libimgutil.a
libimjp81k.a
libimjpcic.a
libimjpcus.a
libimjpdct.a
libimjputyc.a
libimm32.a
libimsinsnt.a
libimskdic.a
libinetcfg.a
libinetcomm.a
libinetmgr.a
libinetmib1.a
libinetpp.a
libinetppui.a
libinfoadmn.a
libinfocomm.a
libinfoctrs.a
libinfosoft.a
libinitpki.a
libinput.a
libinseng.a
libiphlpapi.a
libipmontr.a
libipnathlp.a
libippromon.a
libiprip.a
libiprop.a
libiprtprio.a
libiprtrmgr.a
libipsecsnp.a
libipsecsvc.a
libipsmsnap.a
libipv6mon.a
libipxsap.a
libirclass.a
libisapips.a
libisatq.a
libiscomlog.a
libiscsidsc.a
libisign32.a
libitircl.a
libitss.a
libixsso.a
libiyuv_32.a
libjet500.a
libjpeg.dll.a
libjscript.a
libjsproxy.a
libkbd101.a
libkbd101a.a
libkbd101b.a
libkbd101c.a
libkbd103.a
libkbd106.a
libkbd106n.a
libkbda1.a
libkbda2.a
libkbda3.a
libkbdal.a
libkbdarme.a
libkbdarmw.a
libkbdax2.a
libkbdaze.a
libkbdazel.a
libkbdbe.a
libkbdbene.a
libkbdblr.a
libkbdbr.a
libkbdbu.a
libkbdca.a
libkbdcan.a
libkbdcr.a
libkbdcz1.a
libkbdcz2.a
libkbdcz.a
libkbdda.a
libkbddiv1.a
libkbddiv2.a
libkbddv.a
libkbdes.a
libkbdest.a
libkbdfa.a
libkbdfc.a
libkbdfi.a
libkbdfo.a
libkbdfr.a
libkbdgae.a
libkbdgeo.a
libkbdgkl.a
libkbdgr1.a
libkbdgr.a
libkbdhe220.a
libkbdhe319.a
libkbdhe.a
libkbdheb.a
libkbdhela2.a
libkbdhela3.a
libkbdhept.a
libkbdhu1.a
libkbdhu.a
libkbdibm02.a
libkbdic.a
libkbdindev.a
libkbdinguj.a
libkbdinhin.a
libkbdinkan.a
libkbdinmar.a
libkbdinpun.a
libkbdintam.a
libkbdintel.a
libkbdir.a
libkbdit142.a
libkbdit.a
libkbdjpn.a
libkbdkaz.a
libkbdkor.a
libkbdkyr.a
libkbdla.a
libkbdlk41a.a
libkbdlk41j.a
libkbdlt1.a
libkbdlt.a
libkbdlv1.a
libkbdlv.a
libkbdmac.a
libkbdmon.a
libkbdne.a
libkbdnec95.a
libkbdnec.a
libkbdnecat.a
libkbdnecnt.a
libkbdno.a
libkbdpl1.a
libkbdpl.a
libkbdpo.a
libkbdro.a
libkbdru1.a
libkbdru.a
libkbdsf.a
libkbdsg.a
libkbdsl1.a
libkbdsl.a
libkbdsp.a
libkbdsw.a
libkbdsyr1.a
libkbdsyr2.a
libkbdtat.a
libkbdth0.a
libkbdth1.a
libkbdth2.a
libkbdth3.a
libkbdtuf.a
libkbdtuq.a
libkbduk.a
libkbdur.a
libkbdurdu.a
libkbdus.a
libkbdusa.a
libkbdusl.a
libkbdusr.a
libkbdusx.a
libkbduzb.a
libkbdvntc.a
libkbdycc.a
libkbdycl.a
libkd1394.a
libkdcom.a
libkerberos.a
libkernel32.a
libkeymgr.a
libkorwbrkr.a
libkrnlprov.a
libks.a
libksguid.a
libksuser.a
libktmw32.a
liblangwrbk.a
liblargeint.a
liblicdll.a
liblicmgr10.a
liblicwmi.a
liblinkinfo.a
liblmhsvc.a
liblmmib2.a
liblmrt.a
libloadperf.a
liblocalsec.a
liblocalspl.a
liblocalui.a
liblocationapi.a
liblog.a
libloghours.a
liblogscrpt.a
liblonsint.a
liblpdsvc.a
liblpk.a
liblprhelp.a
liblprmon.a
liblprmonui.a
liblsasrv.a
liblz32.a
libm.a
libmag_hook.a
libmailmsg.a
libmapi32.a
libmapistub.a
libmcastmib.a
libmcd32.a
libmcdsrv32.a
libmchgrcoi.a
libmciavi32.a
libmcicda.a
libmciole32.a
libmciqtz32.a
libmciseq.a
libmciwave.a
libmdhcp.a
libmdminst.a
libmetadata.a
libmf3216.a
libmf.a
libmfc42.a
libmfc42u.a
libmfcsubs.a
libmfplat.a
libmfuuid.a
libmgmtapi.a
libmidimap.a
libmigism.a
libmiglibnt.a
libmimefilt.a
libmingw32.a
libmingwex.a
libmingwthrd.a
libmlang.a
libmll_hp.a
libmll_mtf.a
libmll_qic.a
libmmcbase.a
libmmcndmgr.a
libmmcshext.a
libmmfutil.a
libmmutilse.a
libmobsync.a
libmodemui.a
libmofd.a
libmoldname.a
libmpr.a
libmprapi.a
libmprddm.a
libmprdim.a
libmprmsg.a
libmprui.a
libmqad.a
libmqcertui.a
libmqdscli.a
libmqgentr.a
libmqise.a
libmqlogmgr.a
libmqoa.a
libmqperf.a
libmqqm.a
libmqrt.a
libmqrtdep.a
libmqsec.a
libmqsnap.a
libmqtrig.a
libmqupgrd.a
libmqutil.a
libmsaatext.a
libmsacm32.a
libmsadce.a
libmsadcf.a
libmsadco.a
libmsadcs.a
libmsadds.a
libmsado15.a
libmsadomd.a
libmsador15.a
libmsadox.a
libmsadrh15.a
libmsafd.a
libmsasn1.a
libmscandui.a
libmscat32.a
libmscms.a
libmsctfmonitor.a
libmsctfp.a
libmsdadiag.a
libmsdaosp.a
libmsdaprst.a
libmsdaps.a
libmsdarem.a
libmsdart.a
libmsdatl3.a
libmsdfmap.a
libmsdmo.a
libmsdrm.a
libmsdtclog.a
libmsdtcprx.a
libmsdtcstp.a
libmsdtctm.a
libmsdtcuiu.a
libmsftedit.a
libmsgina.a
libmsgr3en.a
libmsgrocm.a
libmsgsvc.a
libmshtml.a
libmshtmled.a
libmsi.a
libmsident.a
libmsieftp.a
libmsihnd.a
libmsimg32.a
libmsimtf.a
libmsinfo.a
libmsiprov.a
libmsir3jp.a
libmsisip.a
libmslbui.a
libmsls31.a
libmslwvtts.a
libmsmqocm.a
libmsobcomm.a
libmsobdl.a
libmsobmain.a
libmsobshel.a
libmsobweb.a
libmsoe.a
libmsoeacct.a
libmsoert2.a
libmspatcha.a
libmspmsnsv.a
libmsports.a
libmsrating.a
libmsrle32.a
libmssign32.a
libmssip32.a
libmstask.a
libmstime.a
libmstlsapi.a
libmstscax.a
libmsutb.a
libmsv1_0.a
libmsvcirt.a
libmsvcp60.a
libmsvcr100.a
libmsvcr110.a
libmsvcr120.a
libmsvcr120d.a
libmsvcr80.a
libmsvcr90.a
libmsvcr90d.a
libmsvcrt.a
libmsvfw32.a
libmsvidc32.a
libmsvidctl.a
libmsw3prt.a
libmswsock.a
libmsxactps.a
libmsxml3.a
libmsxs64.a
libmsyuv.a
libmtxclu.a
libmtxdm.a
libmtxex.a
libmtxoci.a
libmycomput.a
libmydocs.a
libnarrhook.a
libncobjapi.a
libncprov.a
libncrypt.a
libncxpnt.a
libnddeapi.a
libnddenb32.a
libndfapi.a
libndis.a
libndisnpp.a
libnetapi32.a
libnetcfgx.a
libnetid.a
libnetlogon.a
libnetman.a
libnetoc.a
libnetplwiz.a
libnetrap.a
libnetshell.a
libnetui0.a
libnetui1.a
libnetui2.a
libnewdev.a
libnextlink.a
libnlhtml.a
libnntpadm.a
libnntpapi.a
libnntpsnap.a
libnormaliz.a
libnpptools.a
libnshipsec.a
libntdll.a
libntdsapi.a
libntdsbcli.a
libntevt.a
libntfsdrv.a
libntlanman.a
libntlanui2.a
libntlanui.a
libntlsapi.a
libntmarta.a
libntmsapi.a
libntmsdba.a
libntmsevt.a
libntmsmgr.a
libntmssvc.a
libntoc.a
libntoskrnl.a
libntprint.a
libntshrui.a
libntvdm64.a
libnwprovau.a
liboakley.a
libobjsel.a
liboccache.a
libocgen.a
libocmanage.a
libocmsn.a
libodbc32.a
libodbc32gt.a
libodbcbcp.a
libodbcconf.a
libodbccp32.a
libodbccr32.a
libodbccu32.a
libodbctrac.a
liboeimport.a
liboemiglib.a
libofffilt.a
libogg.dll.a
libole32.a
liboleacc.a
liboleaut32.a
libolecli32.a
libolecnv32.a
liboledb32.a
liboledb32r.a
liboledlg.a
liboleprn.a
libolesvr32.a
libopengl32.a
libosuninst.a
libovprintmondll.a
libp2p.a
libp2pcollab.a
libp2pgraph.a
libpanmap.a
libpautoenr.a
libpchshell.a
libpchsvc.a
libpcwum.a
libpdh.a
libperfctrs.a
libperfdisk.a
libperfnet.a
libperfos.a
libperfproc.a
libperfts.a
libphotowiz.a
libpid.a
libpidgen.a
libpintlcsa.a
libpintlcsd.a
libpjlmon.a
libpng16.dll.a
libpng.dll.a
libpngfilt.a
libpolicman.a
libpolstore.a
libportabledeviceguids.a
libpowrprof.a
libprintui.a
libprofmap.a
libprovthrd.a
libPS5UI.a
libpsapi.a
libpsbase.a
libpschdprf.a
libPSCRIPT5.a
libpsnppagn.a
libpstorec.a
libpstorsvc.a
libpthread.a
libpthread.dll.a
libqasf.a
libqcap.a
libqdv.a
libqdvd.a
libqedit.a
libqmgr.a
libqmgrprxy.a
libqosname.a
libquartz.a
libquery.a
libqutil.a
libqwave.a
libracpldlg.a
librasadhlp.a
librasapi32.a
librasauto.a
libraschap.a
librasctrs.a
librasdlg.a
librasman.a
librasmans.a
librasmontr.a
librasmxs.a
librasppp.a
librasrad.a
librassapi.a
librasser.a
librastapi.a
librastls.a
librcbdyctl.a
librdchost.a
librdpcfgex.a
librdpsnd.a
librdpwsx.a
libregapi.a
libregsvc.a
libregwizc.a
libremotepg.a
librend.a
librepdrvfs.a
libresutil.a
libresutils.a
libriched20.a
librnr20.a
libroutetab.a
librpcdiag.a
librpchttp.a
librpcns4.a
librpcnsh.a
librpcref.a
librpcrt4.a
librpcss.a
librsaenh.a
librsfsaps.a
librshx32.a
librsmps.a
librstrmgr.a
librtm.a
librtutils.a
libruntimeobject.a
librwnh.a
libsafrcdlg.a
libsafrdm.a
libsafrslv.a
libsamlib.a
libsamsrv.a
libsapi.a
libscarddlg.a
libsccbase.a
libsccsccp.a
libscecli.a
libscesrv.a
libschannel.a
libschedsvc.a
libsclgntfy.a
libscredir.a
libscript.a
libscripto.a
libscriptpw.a
libscrnsave.a
libscrnsavw.a
libscrobj.a
libscrptutl.a
libscrrun.a
libsdhcinst.a
libSDL2.a
libSDL2.dll.a
libSDL2_image.dll.a
libSDL2main.a
libSDL2_mixer.dll.a
libsdpblb.a
libseclogon.a
libsecur32.a
libsecurity.a
libsendcmsg.a
libsendmail.a
libsens.a
libsensapi.a
libsenscfg.a
libsensorsapi.a
libseo.a
libseos.a
libserialui.a
libservdeps.a
libserwvdrv.a
libsetupapi.a
libsetupqry.a
libsfc.a
libsfcfiles.a
libsfc_os.a
libsfmapi.a
libshdocvw.a
libshell32.a
libshfolder.a
libshgina.a
libshimeng.a
libshimgvw.a
libshlwapi.a
libshmedia.a
libshscrap.a
libshsvcs.a
libsigtab.a
libsimptcp.a
libsisbkup.a
libskdll.a
libslayerxp.a
libslbcsp.a
libslbiop.a
libslc.a
libslcext.a
libslwga.a
libsmlogcfg.a
libsmtpadm.a
libsmtpapi.a
libsmtpcons.a
libsmtpctrs.a
libsmtpsnap.a
libsmtpsvc.a
libsniffpol.a
libsnmpapi.a
libsnmpcl.a
libsnmpincl.a
libsnmpmib.a
libsnmpsmir.a
libsnmpsnap.a
libsnmpstup.a
libsnmpthrd.a
libsnprfdll.a
libsoftkbd.a
libsoftpub.a
libspcommon.a
libspoolss.a
libsptip.a
libspttseng.a
libsqlsrv32.a
libsqlxmlx.a
libsrchctls.a
libsrchui.a
libsrclient.a
libsrrstr.a
libsrsvc.a
libsrvsvc.a
libssdpapi.a
libssdpsrv.a
libssinc.a
libssp.a
libssp.dll.a
libsspicli.a
libssp_nonshared.a
libsstub.a
libstaxmem.a
libstclient.a
libstdc++.a
libstdc++.dll.a
libstdc++.dll.a-gdb.py
libstdprov.a
libsti.a
libsti_ci.a
libstobject.a
libstorprop.a
libstreamci.a
libstrmfilt.a
libstrmiids.a
libsupc++.a
libsvcext.a
libsvcpack.a
libswprv.a
libsxs.a
libsynceng.a
libsyncui.a
libsysinv.a
libsysmod.a
libsyssetup.a
libt2embed.a
libtapi32.a
libtapi3.a
libtapiperf.a
libtapisrv.a
libtbs.a
libtcpmib.a
libtcpmon.a
libtcpmonui.a
libtdh.a
libtermmgr.a
libtermsrv.a
libthawbrkr.a
libthemeui.a
libtiff.dll.a
libtiffxx.dll.a
libtlntsvrp.a
libtraffic.a
libtrialoc.a
libtrkwks.a
libtsappcmp.a
libtsbyuv.a
libtscfgwmi.a
libtsd32.a
libtshoot.a
libtsoc.a
libturbojpeg.dll.a
libtwext.a
libtxflog.a
libtxfw32.a
libudhisapi.a
libufat.a
libuihelper.a
libulib.a
libumandlg.a
libumdmxfrm.a
libumpnpmgr.a
libUNIDRV.a
libUNIDRVUI.a
libuniime.a
libunimdmat.a
libuniplat.a
libuntfs.a
libupnp.a
libupnphost.a
libupnpui.a
libureg.a
liburl.a
liburlauth.a
liburlmon.a
libusbcamd2.a
libusbd.a
libusbmon.a
libusbport.a
libuser32.a
libuserenv.a
libusp10.a
libutildll.a
libuuid.a
libuxtheme.a
libvbscript.a
libvdsbas.a
libvdsdyndr.a
libvds_ps.a
libvdsutil.a
libvdswmi.a
libverifier.a
libversion.a
libvfw32.a
libvgx.a
libviewprov.a
libvirtdisk.a
libvmx_mode.a
libvorbis.dll.a
libvorbisenc.dll.a
libvorbisfile.dll.a
libvssapi.a
libvss_ps.a
libvsstrace.a
libvsswmi.a
libw32time.a
libw32topl.a
libw3cache.a
libw3comlog.a
libw3core.a
libw3ctrlps.a
libw3ctrs.a
libw3dt.a
libw3ext.a
libw3isapi.a
libw3ssl.a
libw3tp.a
libwab32.a
libwabimp.a
libwamreg.a
libwamregps.a
libwbemcore.a
libwbemupgd.a
libwbemuuid.a
libwdigest.a
libwdmaud.a
libwdsclient.a
libwdsclientapi.a
libwdscore.a
libwdscsl.a
libwdsimage.a
libwdstptc.a
libwdsupgcompl.a
libwdsutil.a
libwebcheck.a
libwebclnt.a
libwebhits.a
libwebpdecoder.dll.a
libwebpdemux.dll.a
libwebp.dll.a
libwebpmux.dll.a
libwecapi.a
libwer.a
libwevtapi.a
libwevtfwd.a
libwiadss.a
libwiarpc.a
libwiaservc.a
libwiashext.a
libwin32spl.a
libwindowscodecs.a
libwinfax.a
libwinhttp.a
libwininet.a
libwinipsec.a
libwinmm.a
libwinpthread.a
libwinpthread.dll.a
libwinrnr.a
libwinscard.a
libwinspool.a
libwinsrv.a
libwinsta.a
libwintrust.a
libwinusb.a
libwkssvc.a
libwlanapi.a
libwlanui.a
libwlanutil.a
libwldap32.a
libwlnotify.a
libwlstore.a
libwmcodecdspuuid.a
libwmi2xml.a
libwmi.a
libwmiaprpl.a
libwmilib.a
libwmiprop.a
libwmisvc.a
libwow64.a
libwow64cpu.a
libwow64mib.a
libwow64win.a
libwpd_ci.a
libws2_32.a
libws2help.a
libwscsvc.a
libwsdapi.a
libwshatm.a
libwshbth.a
libwshcon.a
libwsock32.a
libwtsapi32.a
libx3daudio1_2.a
libx3daudio1_3.a
libx3daudio1_4.a
libx3daudio1_5.a
libx3daudio1_6.a
libx3daudio1_7.a
libx3daudio.a
libx3daudiod1_7.a
libxapofx1_0.a
libxapofx1_1.a
libxapofx1_2.a
libxapofx1_3.a
libxapofx1_4.a
libxapofx1_5.a
libxapofx.a
libxapofxd1_5.a
libxaudio2_8.a
libxinput1_1.a
libxinput1_2.a
libxinput1_3.a
libxinput1_4.a
libxinput9_1_0.a
libxinput.a
libz.dll.a
libzoneoc.a
pkgconfig
txtmode.o

```

</details>

A conan-center hook to lower-case all system libraries on Windows would be useful, imho.

Reply:
soooooooooo, merge? 🙂

Reply:
I think it will be auto-merged, as there are already 3 approvals